securedrop/journalist_templates/_sources_confirmation_final_modal.html

```diff
@@ -0,0 +1,19 @@
+<div id="{{ modal_data.modal_id }}" class="menu-modal-dialog">
+  <a href="#close" class="external"></a>
+  <div id="delete-confirm-menu-dialog">
+    <p>
+      {{ gettext('When the login account for a source is deleted:') }}
```

Comment:
This is consistent with the JI content audit, but I would recommend just saying "account for a source" instead of "**login** account". The phrase "login account" occurs nowhere else (including in this PR, which otherwise just uses "account"), and I would caution against introducing phrases that may introduce additional confusion if they do not recur and are not defined. It's also shorter and IMO sufficiently clear without that qualifier.

Reply:
:heavy_check_mark: concur.